src/lib.rs

```diff
@@ -32,6 +35,36 @@ use zenoh::{
 };
 
 const UATTRIBUTE_VERSION: u8 = 1;
+const THREAD_NUM: usize = 10;
```

Comment:
Should this be configurable by an end-user or is this prone to mis-use if exposed? Certainly we don't want a user to accidentally configure a thread-pool of 100 instead of 10 :sweat_smile: 

Curious to hear your thoughts too @stevenhartley, @sophokles73, @AnotherDaniel

Reply:
Bumping for visibility to @stevenhartley, @sophokles73, @AnotherDaniel